pkg/cmd/tknpac/repository/create.go

```diff
@@ -207,6 +207,14 @@ func getRepoURL(opts *createOptions) error {
 	return fmt.Errorf("no url has been provided")
 }
 
+func cleanupGitURL(url string) string {
```

Comment:
i think my comments i have sent before got lost :\ 

in a nutshell you better use net/url for this : 

```go
	u, err := url.Parse(uu)
	if err != nil { return err }
	fmt.Srintf("%s://%s%s", u.Scheme, u.Host, u.Path)
```